test/integration/serverless/snapshots/timeout-java

```diff
@@ -1,17 +1,16 @@
-[sketch] {"metric":"aws.lambda.enhanced.billed_duration","distributions":null,"tags":["account_id:601427279990","architecture:x86_64","aws_account:601427279990","cold_start:false","dd_extension_version:123","env:integration-tests-env","function_arn:arn:aws:lambda:sa-east-1:601427279990:function:integration-tests-extension-XXXXXX-timeout-java","functionname:integration-tests-extension-XXXXXX-timeout-java","memorysize:1024","region:sa-east-1","resource:integration-tests-extension-XXXXXX-timeout-java","runtime:provided.al2","service:integration-tests-service","version:integration-tests-version"],"dogsketches":[{"ts":XXX,"cnt":XXX,"min":XXX,"max":XXX,"avg":XXX,"sum":XXX,"k":[XXX],"n":[1]}]}
 [sketch] {"metric":"aws.lambda.enhanced.billed_duration","distributions":null,"tags":["account_id:601427279990","architecture:x86_64","aws_account:601427279990","cold_start:true","dd_extension_version:123","env:integration-tests-env","function_arn:arn:aws:lambda:sa-east-1:601427279990:function:integration-tests-extension-XXXXXX-timeout-java","functionname:integration-tests-extension-XXXXXX-timeout-java","memorysize:1024","region:sa-east-1","resource:integration-tests-extension-XXXXXX-timeout-java","runtime:provided.al2","service:integration-tests-service","version:integration-tests-version"],"dogsketches":[{"ts":XXX,"cnt":XXX,"min":XXX,"max":XXX,"avg":XXX,"sum":XXX,"k":[XXX],"n":[1]}]}
-[sketch] {"metric":"aws.lambda.enhanced.duration","distributions":null,"tags":["account_id:601427279990","architecture:x86_64","aws_account:601427279990","cold_start:false","dd_extension_version:123","env:integration-tests-env","function_arn:arn:aws:lambda:sa-east-1:601427279990:function:integration-tests-extension-XXXXXX-timeout-java","functionname:integration-tests-extension-XXXXXX-timeout-java","memorysize:1024","region:sa-east-1","resource:integration-tests-extension-XXXXXX-timeout-java","runtime:provided.al2","service:integration-tests-service","version:integration-tests-version"],"dogsketches":[{"ts":XXX,"cnt":XXX,"min":XXX,"max":XXX,"avg":XXX,"sum":XXX,"k":[XXX],"n":[1]}]}
+[sketch] {"metric":"aws.lambda.enhanced.billed_duration","distributions":null,"tags":["account_id:601427279990","architecture:x86_64","aws_account:601427279990","cold_start:true","dd_extension_version:123","env:integration-tests-env","function_arn:arn:aws:lambda:sa-east-1:601427279990:function:integration-tests-extension-XXXXXX-timeout-java","functionname:integration-tests-extension-XXXXXX-timeout-java","memorysize:1024","region:sa-east-1","resource:integration-tests-extension-XXXXXX-timeout-java","runtime:provided.al2","service:integration-tests-service","version:integration-tests-version"],"dogsketches":[{"ts":XXX,"cnt":XXX,"min":XXX,"max":XXX,"avg":XXX,"sum":XXX,"k":[XXX],"n":[1]}]}
```

Comment:
There are a lot of snapshot changes, are they all legit (since I don't think tests have changed in the PR)

Reply:
I think they are legit changes. Both invocations are considered cold starts which is what we would expect after a timeout. For some reason, with the longer timeout duration we had previously, sometimes the second invocation would be marked as cold_start=false. There might be an actual issue here but I think these new snapshots are accurate.